src/Microsoft.Extensions.Configuration.FileExtensions/FileConfigurationProvider.cs

```diff
@@ -104,7 +84,7 @@ private void Load(bool reload)
         /// </summary>
         /// <exception cref="FileNotFoundException">If Optional is <c>false</c> on the source and a
         /// file does not exist at specified Path.</exception>
-        public override void Load()
+        public override void DoLoad()
```

Comment:
> Well the ideal place we end up here is that providers should no longer need to care about errors, the error handling happens outside of provider code... 

Ok. Should we move the error handling to a utility method we use any time we invoke Load? Or template method pattern is enough?